extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -486,6 +496,36 @@ export class AmpIframe extends AMP.BaseElement {
     });
   }
 
+  /**
+   * Listen for Pym.js messages for 'height' and 'width'.
+   *
+   * @see http://blog.apps.npr.org/pym.js/
+   * @param {MessageEvent} event
```

Comment:
`!MessageEvent`

Reply:
Done in 8dce04812d6005ef4b2a16f45ec6bb1df6c374d3 and 761267de2e42a922353f6392bc8526fdff68eb13